README.md

```diff
@@ -14,20 +14,9 @@ Eve is a set of tools to help us think. Currently, these tools include: a tempor
 
 ### From Source
 
-Before you can install Eve you'll need need LuaJIT, gcc, make, and python. Currently, building from source is only supported on Linux and OSX. On Windows, we've managed to compile Eve in cygwin, but have not gotten it to run yet. Contributions are welcome on this front.
```

Comment:
Users don't need to install LuaJIT manually anymore. It's built by default with Eve.